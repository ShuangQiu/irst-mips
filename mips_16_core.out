10111001110000000000000000101101
