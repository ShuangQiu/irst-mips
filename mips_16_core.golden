10101101100110011000011101011011
